src/mame/intel/imds2.cpp

```diff
@@ -97,7 +97,7 @@ namespace {
 #define IPC_XTAL_Y2     8_MHz_XTAL
 
 // Y1 oscillator of IPC board: 19.6608 MHz
-#define IPC_XTAL_Y1     19.6608_MHz_XTAL
+constexpr auto IPC_XTAL_Y1 = 19.6608_MHz_XTAL;
```

Comment:
If you turn this into a global variable, it needs to have a constant address across an invocation and hence increases static data size.  If it’s only used in a single function, you can make it a local variable.

Reply:
Aren't constexpr variables implicitly inline since C++17?